webapp/src/webusb.tsx

```diff
@@ -200,11 +200,17 @@ function showConnectionSuccessAsync(confirmAsync: ConfirmAsync, willTriggerDownl
 
 function showConnectionFailureAsync(confirmAsync: ConfirmAsync, showDownloadAsFileButton?: boolean) {
     const boardName = getBoardName();
+    const tryAgainText = lf("Try Again");
+    const helpText = lf("Help");
+    const downloadAsFileText = lf("Download as file");
 
     const jsxd = () => (
         <div>
             <div className="ui content download-troubleshoot-header">
-                {lf("We couldn't find your {0}. Here's a few ways to fix that:", boardName)}
+                {lf("We couldn't find your {0}.", boardName)}
+                <br />
+                <br />
+                {lf("Click \"{0}\" for more info, \"{1}\" to try to pair again, or \"{2}\" for drag and drop pairing.", helpText, tryAgainText, downloadAsFileText)}
```

Comment:
Is "pairing" the right word for the drag-and-drop scenario? I wonder if "drag-and-drop install" or something would be more applicable.

Reply:
I was wondering the same thing. "Pairing" doesn't sound quite right to me.